.travis.yml

```diff
@@ -27,4 +40,16 @@ jobs:
     # integration tests
     - src/htsinfer.py --help
     - src/htsinfer.py --debug
+  
+  - stage: push_image
```

Comment:
This won't work. You need to mention, as I mentioned earlier, that different stages _run in parallel_, on different VMs. So if you have a stage that just pushes a workflow, there will be no image to push. So the code to build the image needs to be added to this stage as well.

Reply:
Yeah , I know that we discussed yesterday, about the putting them into a single stage, that's why I wrote in the description I will combine these stages after adding the remaining issues-
- after versioning scheme.
- CI pipeline for automatic PyPI publishing.
Nonetheless I will make all the changes as you proposed and do the things accordingly, and take care of all the issues.